kanidmd/src/lib/actors/v1_read.rs

```diff
@@ -154,6 +162,7 @@ impl QueryServerReadV1 {
             .await
             .and_then(|r| idm_auth.commit(&mut audit).map(|_| r));
 
+        security_info!("Sending auth result -> {{why is this thing so large}}" /*, res*/);
```

Comment:
I assume this is a testing thing? (also all the double-ups of `admin_error` and `ladmin_error`)

Reply:
> I assume this is a testing thing? (also all the double-ups of `admin_error` and `ladmin_error`)

Yeah, I omitted `res` for testing purposes because it was polluting the look of the log which is what I'm trying to get a sense for right now.

The double ups are because I'm just adding my logs in parallel right now, the `ladmin_error`'s will be removed at a later time :)